gcloud/_helpers.py

```diff
@@ -394,6 +394,27 @@ def _to_bytes(value, encoding='ascii'):
         raise TypeError('%r could not be converted to bytes' % (value,))
 
 
+def _from_bytes(value):
```

Comment:
RE: clarity, it follows a pattern of `to` / `from` helpers in this file. But name it whatever you like.